ui/src/app/routes.tsx

```diff
@@ -234,11 +237,27 @@ const routes: AppRouteConfig[] = [
   },
   {
     component: AuditView,
-    exact: true,
     label: 'Audit View',
     path: '/audit',
     title: 'Audit View',
   },
+  {
+    component: AuditRules,
+    exact: true,
+    path: '/audit/rules',
+    title: 'Audit Rules',
+  },
+  {
+    component: AuditHosts,
+    exact: true,
+    path: '/audit/hosts',
+    title: 'Audit Hosts',
+  },
+  {
+    component: AuditRule,
+    path: '/audit-rule/:id',
```

Comment:
```suggestion
    path: '/audit-rules/:id/details',
```

Reply:
/audit-rule/:id is the defauult route for the details page